vimeo-networking/src/main/java/com/vimeo/networking/model/InteractionCollection.java

```diff
@@ -56,6 +56,10 @@
     protected Interaction mBuy;
 
     @Nullable
+    @SerializedName("purchase")
+    protected Interaction mPurchase;
```

Comment:
Why protected?

Reply:
Same as above -- because everything else is protected! :)